### PR TITLE
tests: ++coverage

### PR DESCRIPTION
**Pull request by mgrudzinska**
without any `extra`:

lcov_2606

### CODE REVIEW

File: test/testText.cpp [hunk 1]
```diff
@@ -129,10 +129,10 @@ TEST_CASE("Text Basic", "[tvgText]")
     REQUIRE(Text::load(TEST_DIR"/Arial.ttf") == tvg::Result::Success);
     REQUIRE(text->font("Arial", 80) == tvg::Result::Success);
 
-    REQUIRE(text->text("THORVG Text") == tvg::Result::Success);
-    REQUIRE(text->text("ABCDEFGHIJIKLMOPQRSTUVWXYZ") == tvg::Result::Success);
-    REQUIRE(text->text("") == tvg::Result::Success);
     REQUIRE(text->text(nullptr) == tvg::Result::Success);
+    REQUIRE(text->text("") == tvg::Result::Success);
+    REQUIRE(text->text("ABCDEFGHIJIKLMOPQRSTUVWXYZ") == tvg::Result::Success);
+    REQUIRE(text->text("THORVG Text") == tvg::Result::Success);
```
Comment (tinyjin) [inline test/testText.cpp:135]: Wondering how this changes improve test coverage. I see the code lines are the same.

**Review by tinyjin — APPROVED** [review]

File: test/testAnimation.cpp [hunk 2]
```diff
@@ -276,4 +291,23 @@ TEST_CASE("Animation Segment", "[tvgAnimation]")
     REQUIRE(Initializer::term() == Result::Success);
 }
 
+#if THORVG_LOTTIE_EXPRESSIONS_SUPPORT
```
Comment (hermet) [inline test/testAnimation.cpp:294]: @mgrudzinska Let's remove #if THORVG_LOTTIE_EXPRESSIONS_SUPPORT

File: test/testText.cpp (hunk 1 above)
Comment (mgrudzinska) [reply test/testText.cpp:135]: from tvgText.h text(): 
```
        free(this->utf8);
        if (utf8) this->utf8 = strdup(utf8);
```
text is overwritten , so if the last one passed was 'nullptr' not much was really tested in the ttf loader

**Review by hermet — APPROVED** [review]

**Comment by hermet** [comment]
@miragrudzinska thanks, please update the figure => https://github.com/thorvg/thorvg/wiki/Unit-Tests, (+ no need to add "static=true" option)

**Pull request merged**